pkg/alerts/alerts_types.go

```diff
@@ -190,6 +190,24 @@ type SyntheticsCondition struct {
 	MonitorID  string `json:"monitor_id,omitempty"`
 }
 
+// LocationFailureCondition represents a location-based failure condition.
+type LocationFailureCondition struct {
+	ID                        int                            `json:"id,omitempty"`
+	Name                      string                         `json:"name,omitempty"`
+	Enabled                   bool                           `json:"enabled"`
+	RunbookURL                string                         `json:"runbook_url,omitempty"`
+	MonitorID                 string                         `json:"monitor_id,omitempty"`
+	Entities                  []string                       `json:"entities,omitempty"`
+	Terms                     []LocationFailureConditionTerm `json:"terms,omitempty"`
+	ViolationTimeLimitSeconds int                            `json:"violation_time_limit_seconds,omitempty"`
+}
+
+// LocationFailureConditionTerm represents a single term for a location-based failure condition.
```

Comment:
Should we include a note here about how to configure these?  In terms of how `Threshold` counts location failures?

Reply:
Yes, good point.  The docs in the API explorer are not quite complete, since what is meant by `integer` in some cases really means, the number of seconds in intervals of 1, 2, 4, 8 hours.  I have a slack message with some more accurate docs that I could transpose here.  I was wondering about how the official docs get updated, but in lieu of proper docs, at least notes on the method would be good.

Reply:
I've added a bit of information in the couple of places that were confusing like this one.  I'd been wondering about validating the input before we make the call, or if we instead just rely on good error messaging back from the API to return to the caller.  Maintaining that validation would put us in the middle of updating the code each time there were a change to the API, which doesn't sound great, but would provide us the opportunity to give the caller perhaps a fast and perhaps more informative error in the case where the API error returned is less than ideal.